src/create-entity-api.ts

```diff
@@ -164,6 +210,16 @@ export function createEntityApi<
           }
         }),
 
+        /**
+         * Creates a query endpoint that requests `GET /{baseEndpoint}/{id}` to fetch single entity data by ID.
+         * Can accept optional request params which implements `EntityRequest` instance.
+         * Request shape can be extended by `entityGetRequestConstructor` property in createEntityApi.
```

Comment:
```suggestion
        * Accepts request params described by `entityGetRequestConstructor`
```

Reply:
@ipakhomov should we change this text in README too?